src/util/substrateProviders.ts

```diff
@@ -52,8 +65,32 @@ export const customTypes: CustomTypes = {
   [CHAIN_2]: types[CHAIN_2]
 };
 
+export const bridgeIds: GenericKeyStringConfigs = {
+  [CHAIN_1]: 'rlto',
+  [CHAIN_2]: 'mlau'
+};
+
 export const customHashers: CustomHashers = {
   [CHAIN_2]: hasherH512
 };
 
+export const ss58Formats: GenericKeyNumberConfigs = {
+  [CHAIN_1]: 48,
+  [CHAIN_2]: 60
+};
```

Comment:
I'm not sure if this is the most convenient way of providing chain-specific configuration, why not group all parameters per chain like so:
```
export const knownChains = {
  'Rialto': {
     ss58Prefix: 48,
     customHasher: null,
     bridgeId: 'rlto',
     customTypes: types.Rialto,
     provider: 'wss://wss.rialto.brucke.link',
  },
  'Millau': {
     ...
  },
};
```

We can introduce parametrising (via env-variables) this later on when we know all of the parameters, because now it becomes unmaintainable.